### DISCUSSION TITLE
[REQUEST] Support for classic theme with hidden labels and support for briefcase.

### DISCUSSION THREAD

**Discussion by OrthodoxWindows** [valinet/ExplorerPatcher#1072] [Ideas]
Contrary to what the title indicates it is not asking for the short term, but in the long run; I simply pose that now because it would be unappropriated for you these requests once the forum is closed.

During my question about the  issue with the labels off in taskbar and advanced mitigations for classic theme enabled (https://github.com/valinet/ExplorerPatcher/discussions/900), you indicated to me that it was not possible to set the buttons space on the taskbar as to the labels was hidden with the classic theme activated. It turns out that I got by chance (in the latest version), a interesting bug by activating the weater widget (more precisely, by moving it Left/top). I took a bug screenshot (below), but I did not manage to reproduce it (even employing the same path, the bug disappeared). may be mistaken, but this bug may prove that it is technically possible to force the buttons to use a defined space, without the labels, and with the classic theme mitigations.

![UkroMS0Unh](https://user-images.githubusercontent.com/100134023/158204941-55781ecf-0d83-46c2-accc-b663850f2057.png)

For the briefcase, this is a synchronization functionality that was present from Windows NT4 to the first versions of Windows 10, which was then deleted (https://winaero.com/the-briefcase-feature-is-removed-in-windows-10-redstone-2/). 
The main problem is that I feel that the DLL/EXE concerned is SyncApp.exe, which seems to have been removed. However, it is possible that the code is included elsewhere, I do not know too much. I suggest that if you ever are a day interested by this feature.

In any case, thank you, and thank you again for this great project that you have led incredibly.

**Comment by valinet** [comment]
I never said it was impossible. Everything is possible. I said I don't know how to do it. Of course it is technically possible. Why wouldn't it be? It's software, it can do whatever one codes it to do.
